resources/views/front-end/photo_album_list.blade.php

```diff
@@ -0,0 +1,82 @@
+<head>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script> 
+<script src="../js/photoalbum.js"></script>
+</head>
```

Comment:
Voor inline styles kan je beter `@push('styles')` gebruiken, ook zou ik dit als dependency in package.json zetten en dan gewoon lokaal inladen ipv een CDN.

Reply:
Nice! Is het wel nodig om font awesome te gebruiken? We hebben namelijk al een ander icon font.

Reply:
In gebruik het nu voor de spinner in de button. Niet per see nodig maar laat goed zien dat de pagina bezig is.

Reply:
Maar kan dat niet ook met het icon font dat toch al gebruikt wordt? Lijkt me zonde om heel font awesome in te laden voor één spinner.

Reply:
Waarschijnlijk wel. Welk font is dat?
Zou je ook kunnen beschrijven hoe ik andere dependencies zoals Photoswipe lokaal inlaad?

Reply:
Het icon font is "ionicons", lokaal inladen is eigenlijk niet heel belangrijk denk ik.

Reply:
Ik heb even gekeken naar ionicons, dit zijn enkel icons en bevat geen spinners. daarvoor is volgensmij het volledige ionic framework voor nodig. Ik scharp dit spinnertje even. is ook niet erg belangrijk